src/reps/string.js

```diff
@@ -39,17 +41,18 @@ const StringRep = React.createClass({
       config.style = style;
     }
 
+    if (this.props.useQuotes) {
+      text = escapeString(text);
+    }
     if (member && member.open) {
-      return span(config, "\"" + text + "\"");
+      // Nothing.
+    } else if (this.props.cropLimit) {
+      text = cropString(text, this.props.cropLimit);
+    } else {
+      text = sanitizeString(text);
     }
```

Comment:
I think we could improve these consecutive if blocks

Could we rewrite it so that we have a first step which cleans up the string, either using escapeString or sanitizeString depending on useQuotes. After that we perform the cropping if needed. 

Also might make clearer what is the impact of member && member.open. Not calling sanitizeString in this case seemed like a flaw of the previous implementation? Why would we ever want to allow characters that can't be displayed? Maybe @nchevobbe will know more. My guess would be that if member && member.open, we should do the same as for "useQuotes"